app/src/components/map/googlemap/promise/PromiseStors.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Api from '../../../../api/Api';
+import StoresApi from '../../../../api/stores/StoresApi';
+
+export async function getStorsInformation() {
+  const stores = await Api.StoresApi.list();
+  for (const store of stores) {
+    if (store.id != 1) {
+      const localisation = store.attributes.localization;
+      store.attributes.localization = localisation.split(',');
```

Comment:
*getStoresInformation ici aussi, pareil titre du fichier *PromiseStores
Et comme c'est de la récup de données qui peut être générique et pas un composants, déplace plutot ton dossier dans src/promise

Reply:
Et du coup ce qu'on voudrait c'est d'envoyer la localization en dégeu comme dans la BD et la traiter au dernier moment, en soit on aurait pu les traiter avant mais il aurait fallu faire ca dès la partie BD, API ou dans nos entités, donc envoi les loc pas encore splittées

Reply:
Et pareil au lieu de faire un for if fais plutot un filter en JS
Du genre 
const filteredStores = stores.filter( (store) => store.id != 1 );